src/Razor/src/Microsoft.AspNetCore.Razor.ExternalAccess.OmniSharp/Project/IOmniSharpProjectSnapshotManagerChangeTrigger.cs

```diff
@@ -0,0 +1,9 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT license. See License.txt in the project root for license information.
+
+namespace Microsoft.AspNetCore.Razor.ExternalAccess.OmniSharp.Project;
+
+public interface IOmniSharpProjectSnapshotManagerChangeTrigger
+{
+    public abstract void Initialize(OmniSharpProjectSnapshotManager projectManager);
```

Comment:
Doesn't this need to not have `public abstract` now? Wouldn't have thought this would compile.

Reply:
Huh, you're right that one might expect that to fail compilation (and yet it obviously has not). I removed the "public abstract".